lib/rules/newline-after-var.js

```diff
@@ -120,20 +122,24 @@ module.exports = {
         }
 
         /**
+        * Gets the last line of a group of consecutive comments
+        * @param {number} commentStartLine The starting line of the group
+        * @returns {number} The number last commented line of the group
```

Comment:
This description sounds a bit off, probably something got lost with some typos?
How about something like: "`The number of the last comment line of the group`"